app/assets/stylesheets/components/_figure.scss

```diff
@@ -16,24 +16,19 @@
   }
 }
 
-.app-c-figure__figcaption-wrapper {
-  @include media(tablet) {
-    box-sizing: border-box;
-    float: left;
-    padding-left: $gutter-half;
-    width: 50%;
-  }
-}
-
 .app-c-figure__figcaption {
   @include core-16;
 
-  width: 100%;
 
   @include media(tablet) {
     @include core-14;
 
     display: block;
     vertical-align: top;
+
+    box-sizing: border-box;
```

Comment:
Do we need to use the border-box mixins for cross browser support still?

Reply:
I don't know, I took the approach of fixing the bug with as little change/interference as possible.

Reply:
Looks like the browsers we support should work fine from what I can see.